elements/tcpudp/udpipencap.cc

```diff
@@ -43,15 +43,14 @@ UDPIPEncap::configure(Vector<String> &conf, ErrorHandler *errh)
 {
     IPAddress saddr;
     uint16_t sport, dport;
-    bool cksum;
     String daddr_str;
 
     if (Args(conf, this, errh)
 	.read_mp("SRC", saddr)
 	.read_mp("SPORT", IPPortArg(IP_PROTO_UDP), sport)
 	.read_mp("DST", AnyArg(), daddr_str)
 	.read_mp("DPORT", IPPortArg(IP_PROTO_UDP), dport)
-	.read_p("CHECKSUM", BoolArg(), cksum)
+	.read_p("CHECKSUM", BoolArg(), _cksum)
```

Comment:
Thanks for this patch! But the code is written to ensure that the UDPIPEncap variables are *only* changed if the configuration as a whole succeeds. I'm going to check in a different fix that does that.